docs/src/Box.doc.js

```diff
@@ -651,23 +803,38 @@ function ButtonFlyoutExample() {
     </>
   );
 }`}
-  />,
-);
-
-card(
-  <Example
-    description={`
-It's possible to use box with external elements using the css \`z-index\` property by capturing those values in controlled objects. The example below shows using a \`FixedZIndex\` for a value that comes from somewhere else, and a \`CompositeZIndex\` to layer the Box on top of it.
-  `}
-    id="zindex"
-    name="ZIndex"
-    defaultCode={`
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`It's possible to use Box with external elements using the css \`z-index\` property by capturing those values in controlled objects. The example below shows using a \`FixedZIndex\` for a value that comes from somewhere else, and a \`CompositeZIndex\` to layer the Box on top of it.`}
+      title="Z-Index"
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 function Example() {
   const HEADER_ZINDEX = new FixedZIndex(100);
   const zIndex = new CompositeZIndex([HEADER_ZINDEX]);
   return <Box color="blue" width={60} height={60} zIndex={zIndex} />
 }
 `}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection
+    name="Related"
```

Comment:
This should probably also include TapArea, Sticky, Card, Column, Layer, Letterbox, Mask

Reply:
I think we should limit Related to 3 or 4 options, so we don't totally overwhelm people. It should be the 3-4 closest matches, so maybe TapArea and Sticky? Both things that are more utility components, and not presentational

Reply:
I agree in general but think it's fine to break that rule for Box, since it's so foundational. I don't think it's really clear when a user would want Card, Column, or Letterbox

(this can def be addressed in a future PR)